packages/api/cms-api/src/user-permissions/user-permissions.types.ts

```diff
@@ -38,6 +38,7 @@ export interface UserPermissionsUserServiceInterface {
 
 export interface UserPermissionsOptions {
     availableContentScopes?: ContentScope[] | (() => Promise<ContentScope[]> | ContentScope[]);
+    getLabelForContentScope?: (contentScope: ContentScope) => string;
```

Comment:
Is there a use case for making this async? Maybe loading the label based on an ID in the content scope? Or should the label be part of the content scope then?